Libraries/Text/RCTTextField.m

```diff
@@ -112,6 +113,85 @@ - (void)setPlaceholder:(NSString *)placeholder
   RCTUpdatePlaceholder(self);
 }
 
+- (UIColor *)tintColor
+{
+  return super.tintColor;
+}
+
+- (void)setTintColor:(UIColor *)tintColor
+{
+  super.tintColor = tintColor;
+}
+
+- (UIColor *)clearButtonTintColor
+{
+  return _clearButtonTintColor;
+}
+
+- (void)setClearButtonTintColor:(UIColor *)clearButtonTintColor
+{
+    _clearButtonTintColor = clearButtonTintColor;
+}
+
+static UIButton* findClearButton(RCTTextField *self)
```

Comment:
Make this a private class method: `+ (UIButton *)_findClearButton` (cc @nicklockwood)